style/styles.css

```diff
@@ -171,6 +171,59 @@ tbody td {
     padding: 27px 50px;
 }
 
+input[type="checkbox"] {
+    width: 25px; /* Adjust the size */
+    height: 25px; /* Adjust the size */
+    appearance: none; /* Remove default checkbox styling */
+    background-color: #79C8A3; /* Box color */
+    border: 2px solid #309C68; /* Border color */
+    border-radius: 4px; /* Optional: rounded corners */
+    cursor: pointer;
+    outline: none;
+    position: relative;
+}
+
+/* Style for when checkbox is checked */
+input[type="checkbox"]:checked {
+    background-color: #309C68; /* Green with 40% opacity */
+}
+
+input[type="checkbox"]:checked::after {
+    content: '✖'; /* Unicode character for checkmark */
+    color: black; /* Checkmark color */
+    font-size: 20px; /* Size of the checkmark */
+    position: absolute; /* Position the checkmark */
+    top: 0; /* Center vertically */
+    left: 2px; /* Center horizontally */
+}
+
+.buttonSelectStyle {
+    width: 144px;
```

Comment:
Those are a lot of comments.

Reply:
Unicode is cool!

Reply:
dispatchEvent() method allows you to trigger events programmatically on a DOM element. This means you can simulate user actions like clicks, key presses, or any custom events you define.